plugins/com.gwtplugins.gdt.eclipse.suite/plugin.xml

```diff
@@ -109,7 +109,7 @@
       <!-- GWT Super DevMode Launch shortcut when using GWT -->
       <shortcut
             label="GWT Development Mode with Jetty"
-            description="Launch GWT legacy development mode with embedded Jetty server (Super Development Mode)"
```

Comment:
I don't understand why this ever said legacy to begin with, since the line also says super dev mode at the end?